x/emissions/keeper/genesis.go

```diff
@@ -1086,6 +1137,25 @@ func (k *Keeper) ExportGenesis(ctx context.Context) (*types.GenesisState, error)
 		previousForecastRewardFraction = append(previousForecastRewardFraction, &topicIdActorIdDec)
 	}
 
+	/* bug in genesis export, previousForecasterScoreRatio is not correct type in genesis.proto
```

Comment:
can't add as another type and decode here as correct type...?

Reply:
no it's a separate bug completely and I felt was out of scope for this PR. See PROTO-2349 for more details